src/winetricks

```diff
@@ -7158,11 +7158,20 @@ w_metadata depends apps \
 
 load_depends()
 {
-    w_download https://www.dependencywalker.com/depends22_x86.zip 03d73abba0e856c81ba994505373fdb94a13b84eb29e6c268be1bf21b7417ca3
-    w_try_unzip "${W_TMP}" "${W_CACHE}"/depends/depends22_x86.zip
-    w_try cp "${W_TMP}"/depends.* "${W_WINDIR_UNIX}"
-    # depends.exe uses mfc42
-    w_call mfc42
+    if [ "${W_ARCH}" = "win64" ]; then
+        w_download https://www.dependencywalker.com/depends22_x64.zip 35db68a613874a2e8c1422eb0ea7861f825fc71717d46dabf1f249ce9634b4f1
+        w_try_unzip "${W_TMP}" "${W_CACHE}"/depends/depends22_x64.zip
+        w_try cp "${W_TMP}"/depends.* "${W_WINDIR_UNIX}"
+        w_download_to  depends "https://www.dlldownloader.com/mfc42-dll/download/f0c0df5da0fdb6305017c5dec1099a94/07c410623c40f250d422e5801cc57855/?c=bEhCdUZzZjg5T0lYWjU5a3lNU0gyUT09" f3ce67a68b650f33a6b1ff21a0bff1dba75282e4d4a0e4ba72f7c4a336fc4be5 mfc42.zip
```

Comment:
Please don't use links to these sketchy DLL sites. Ideally we'd use a redistributable from microsoft.com. As a fallback, some verbs get dlls from other projects that ship the needed dll (e.g., firefox with d3dcompiler_47.dll).

I haven't tried, but maybe it's available in the win10 iso:
https://www.microsoft.com/en-us/software-download/windows10ISO

Reply:
I checked the iso image, but unfortunately did not find a corresponding file. I have

1. downloaded the iso file for 64 bit Windows 10
2. installed `cabextract
3. executed the following commands
```
sudo mount -o loop Win10_22H2_EnglishInternational_x64v1.iso Win10_22H2_EnglishInternational_x64v1 
find Win10_22H2_EnglishInternational_x64v1 -iname '*mfc*'
<empty list>
```
4. existing archives checked
```
find Win10_22H2_EnglishInternational_x64v1 -iname '*.cab'
Win10_22H2_EnglishInternational_x64v1/sources/sxs/Microsoft-Windows-InternetExplorer-Optional-Package~31bf3856ad364e35~amd64~en-GB~.cab
Win10_22H2_EnglishInternational_x64v1/sources/sxs/microsoft-windows-internetexplorer-optional-package~31bf3856ad364e35~amd64~~.cab
Win10_22H2_EnglishInternational_x64v1/sources/sxs/microsoft-windows-netfx3-ondemand-package~31bf3856ad364e35~amd64~~.cab
Win10_22H2_EnglishInternational_x64v1/sources/uup/metadata/desktoptargetcompdb_neutral.xml.cab
```
5. existing archives unpacked
```
mkdir tmp && cd tmp
find ../Win10_22H2_EnglishInternational_x64v1/ -iname '*.cab' | xargs cabextract
find -iname '*mfc*.dll'
<empty list>
```
6. existing archives checked 
```
find -iname '*.cab'
<empty list>
```
 
I currently have no idea how to get the desired file from the ISO image.

Reply:
That's unfortunate.

If Microsoft doesn't make it available, though, that doesn't mean we can use an unlicensed copy. I.e., this is a WONTFIX, unless a legal download is available.